pkg/dogstatsd/enrich_test.go

```diff
@@ -16,13 +16,15 @@ func returnEmptyTags() []string {
 	return []string{}
 }
 
+// TODO(remy): add an unit test using the serverless mode and validate that there is no Host set
```

Comment:
Do we want to add the unit test before merging ?

Reply:
I've added a unit test for that but for now we have to run it manually (since it needs a special build tags, we can't just run it while running the regular agent ones). However, I've a following task of integrations the build in the pipeline, invoke tasks, etc.
I'll add the unit tests run at this point if that sounds good to you